src/arch/arm/asm.h

```diff
@@ -9,10 +9,17 @@
 
 #if defined(__APPLE__) || defined(__MACH__)
```

Comment:
If I understand correctly, Apple Silicon chips only support AArch64. Why do we need arm32?

Reply:
Well. Some older iPhones are arm32.